pages/stable-version.html

```diff
@@ -92,25 +92,25 @@ <h1>Stable version 2025.07</h1>
 
             <h4 id="releasenotes"><a href="#releasenotes">Release notes</a></h4>
             <!--<p><strong>WARNING:</strong> Remember to review the release notes! They contain valuable information and, depending on your current version, required procedures to update to the most recent one! Make sure you read the release notes for all the versions between the one you are using now, and 2024.10!</strong></p>-->
-            <p><strong>WARNING:</strong> Remember to review the release notes! They contain valuable information and, for the update to 2024.10, remember that this version is changing the base operating system and moves the product to a containers only release for both Uyuni Server and the Proxies!</strong></p>
+            <p><strong>WARNING:</strong> Remember to review the release notes! They contain valuable information and, since the update to 2024.10, remember that this version was changing the base operating system and moved the product to a containers only release for both Uyuni Server and the Proxies!</strong></p>
```

Comment:
```suggestion
            <p><strong>WARNING:</strong> Remember to review the release notes! Since release 2025.10 Uyuni is released for a different host OS for both Uyuni Server and the Proxies!</strong></p>
```